### PR TITLE
WP 4.7.4, Author Chat, IG Extras, FCM & minor fixes / features

### LINKED ISSUE #260: Always copy media in translated pages
When a site is created, the checkboxes to copy media attachments to translated sites are automatically checked.
However, when editing a page, these checkboxes are not checked:
unbenannt

We should always synchronize media attachments between languages -> hide the checkbox and have it always checked.

related to #244

### LINKED ISSUE #305: Disable notification emails
I am getting annoyed by WordPress' mails for password lost/changed.

There is a plugin that disables it: https://wordpress.org/plugins/disable-new-user-notifications/

Any objections?

### LINKED ISSUE #398: Move Hilfe-Page (Q&A + documents) into CMS
Could we implement something like an "Open in a new Tab" Button for hilfe.integreat-app.de ~~and web.integreat-app.de~~ inside the CMS (menu)? 

This is a desirable feature of Neuss.

EDIT ms: Only the hilfe-button is needed.

### LINKED ISSUE #451: Permission "Delete User" should be in the role "Verwalter"
Please add the permission to remove users from a CMS to the role of the "Verwalter". Thought that would be already in, but it isn't.

### PR DESCRIPTION
**Pull request by sven15**
* add delete_users to verwalter, fixes #451 (#487)
* added link to https://hilfe.integreat-app.de/q2a, fixes #398 (#489)
* lock wpml media checkboxes, fixes #260 (#488)
* disable new user notification emails, fixes #305 (#490)
* bugfix of user rights #495, #496
* bugfix for cloning sites #494 
* translation files for content loader #425 
* simplify UI #403, #406 
* added Lehrstellenradar as parameter for app #503 
* strip all formatting when pasting into editor #243 
* disable all automatic updates
* disable edit of permalinks and update automatically #328 
* delete CL attachments when deactivating for a page #515 
* added zip code to ig-extras #513
* Author Chat #507 
* WP 4.7.4

@FritjofK, @dkehne, @janiner256  Please do some thorough testing on https://cms-test.integreat-app.de

### CODE REVIEW

**Comment by sven15** [comment]
Cannot be merged without fixing #494 and #495

**Comment by sven15** [comment]
Waiting for review and approval of #514

**Comment by mschrimpf** [comment]
Please give a more descriptive name to the PR, the current title applies to almost any PR we ever had ;)

**Comment by sven15** [comment]
@mschrimpf yes I know. But we've got so many features in it that there is no fitting title. We would have to start giving version numbers or something like that.

**Comment by mschrimpf** [comment]
@sven15 how about a list of the major changes? see e.g. #449 
You can still exclude the minor points and list them in the description.

**Comment by Ponitac** [comment]
#514 is completed. So further review for this pull required?

**Comment by sven15** [comment]
Yes, please @Ponitac

File: wp-activate-plugins.sh [hunk 1]
```diff
@@ -1,38 +1,47 @@
-#!/usr/bin/env bash
+#! \usr \bin \env bash
```
Comment (mschrimpf) [inline wp-activate-plugins.sh]: Hier gehören front slashes und keine Leerzeichen hin

File: wp-activate-plugins.sh [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+"vendor/wp-cli/wp-cli/bin/wp" plugin deactivate --network --all
```
Comment (mschrimpf) [inline wp-activate-plugins.sh]: Zeile doppelt (mit 3)

File: wp-content/plugins/ig-content-loader-instance/ig-content-loader-instance-de_DE.po [hunk 3]
```diff
@@ -0,0 +1,8 @@
+msgid "Page from other instance"
+msgstr "Seite aus Fremdinstanz"
+
+msgid "Select city"
+msgstr "Bitte Kommune ausw&auml;hlen"
+
+msgid "Please select"
```
Comment (mschrimpf) [inline wp-content/plugins/ig-content-loader-instance/ig-content-loader-instance-de_DE.po]: Fehlt hier nicht noch das "Please select (insert nothing)"? Oder kommt das in eine andere .po?

File: wp-activate-plugins.sh (hunk 2 above)
Comment (sven15) [inline wp-activate-plugins.sh]: it's a "deactivate" not activate?

**Comment by sven15** [comment]
Thanks for the findings @mschrimpf

**Pull request merged**